src/crafting.cpp

```diff
@@ -231,7 +263,8 @@ void player::craft()
     int batch_size = 0;
     const recipe *rec = select_crafting_recipe( batch_size );
     if (rec) {
-        if ( crafting_allowed() ) {
+        if ( crafting_allowed( *rec ) )
```

Comment:
because I can't figure out a reasonable process to astyle just one block of code. when I run astyle on a block, the rules work differently than running it on the whole file, and when I run it on the whole file it takes forever to cherry pick my block for committing.